internal/dicts/wordmap.go

```diff
@@ -32,9 +32,17 @@ func (m *WordMap) AddFrom(other WordMap) {
 }
 
 // Add a single mapping from one word to another to the WordMap.
+//
+// This function panics if an empty string is provided as first argument.
 func (m *WordMap) AddOne(from, to string) {
-  m.from = append(m.from, strings.TrimSpace(strings.ToLower(from)))
-  m.to = append(m.to, strings.TrimSpace(strings.ToLower(to)))
+  fromValue := strings.TrimSpace(strings.ToLower(from))
+  toValue := strings.TrimSpace(strings.ToLower(to))
+  if fromValue == "" {
```

Comment:
Now, if one does `m.AddOne("foo", "")` and the `WordMap` is [`Invert`ed](https://github.com/ericcornelissen/wordrow/blob/a4b26e23af0b26f9d6d4c939a34f836e9dff941c/internal/dicts/wordmap.go#L77) there will still be a "from" value that is an empty string, namely in the mapping `"" -> "foo"` :worried: